src/windows.zig

```diff
@@ -58,7 +58,7 @@ pub usingnamespace ntdll;
 pub const user32 = windows.user32;
 pub const advapi32 = windows.advapi32;
 
-pub const INVALID_FILE_ATTRIBUTES = -1;
+pub const INVALID_FILE_ATTRIBUTES: u32 = std.math.maxInt(u32);
```

Comment:
is this right

Reply:
Yes, the return value is a u32 as per ms docs, but using -1 is out of range so the comparison results in comptime false.

fun fact, it's an intended feature of zig for this mistake to not be a compiler error... https://github.com/ziglang/zig/issues/17853